pkg/backend/filestate/state.go

```diff
@@ -179,12 +179,14 @@ func (b *localBackend) getCheckpoint(stackName tokens.Name) (*apitype.Checkpoint
 	return stack.UnmarshalVersionedCheckpointToLatestCheckpoint(m, bytes)
 }
 
-func (b *localBackend) saveStack(name tokens.Name, snap *deploy.Snapshot, sm secrets.Manager) (string, error) {
+func (b *localBackend) saveCheckpoint(
+	name tokens.Name, checkpoint *apitype.VersionedCheckpoint) (string, string, error) {
```

Comment:
Consider adding either a doc string above this method,
or using named returns to indicate what the return values are.

```suggestion
	name tokens.Name, checkpoint *apitype.VersionedCheckpoint) (backupFile, file string, _ error) {
```

Without that, it requires digging into the body to understand what this returns.

Reply:
👍 sure